federation/ldap/src/main/java/org/keycloak/storage/ldap/mappers/membership/group/GroupLDAPStorageMapperFactory.java

```diff
@@ -198,6 +198,12 @@
                 .helpText("If this flag is true, then during sync of groups from LDAP to Keycloak, we will keep just those Keycloak groups, which still exists in LDAP. Rest will be deleted")
                 .type(ProviderConfigProperty.BOOLEAN_TYPE)
                 .defaultValue("false")
+                .add()
+                .property().name(GroupMapperConfig.LDAP_GROUPS_PATH)
+                .label("Groups Path")
+                .helpText("Keycloak group path the LDAP groups are added to")
```

Comment:
Is it possible to change tooltip to contain more details? I would use something like: _Keycloak group path the LDAP groups are added to. For example if value '/Applications/App1' is used, then LDAP groups will be available in Keycloak under group 'App1', which is child of top level group 'Applications'. The default value is '/' so LDAP groups will be mapped to the Keycloak groups at the top level. The configured group path must already exists in the Keycloak when creating this mapper._

Reply:
Sure. If you don't mind then I'll use your tooltip text as-is 👍